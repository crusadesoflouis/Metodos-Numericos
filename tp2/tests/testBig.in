full/s1/1.pgm, 1,
full/s1/2.pgm, 1,
full/s1/3.pgm, 1,
full/s1/4.pgm, 1,
full/s1/5.pgm, 1,
full/s2/1.pgm, 2,
full/s2/2.pgm, 2,
full/s2/3.pgm, 2,
full/s2/4.pgm, 2,
full/s2/5.pgm, 2,
full/s3/1.pgm, 3,
full/s3/2.pgm, 3,
full/s3/3.pgm, 3,
full/s3/4.pgm, 3,
full/s3/5.pgm, 3,
full/s4/1.pgm, 4,
full/s4/2.pgm, 4,
full/s4/3.pgm, 4,
full/s4/4.pgm, 4,
full/s4/5.pgm, 4,
full/s5/1.pgm, 5,
full/s5/2.pgm, 5,
full/s5/3.pgm, 5,
full/s5/4.pgm, 5,
full/s5/5.pgm, 5,
full/s6/1.pgm, 6,
full/s6/2.pgm, 6,
full/s6/3.pgm, 6,
full/s6/4.pgm, 6,
full/s6/5.pgm, 6,
full/s7/1.pgm, 7,
full/s7/2.pgm, 7,
full/s7/3.pgm, 7,
full/s7/4.pgm, 7,
full/s7/5.pgm, 7,
full/s8/1.pgm, 8,
full/s8/2.pgm, 8,
full/s8/3.pgm, 8,
full/s8/4.pgm, 8,
full/s8/5.pgm, 8,
full/s9/1.pgm, 9,
full/s9/2.pgm, 9,
full/s9/3.pgm, 9,
full/s9/4.pgm, 9,
full/s9/5.pgm, 9,
full/s10/1.pgm, 10,
full/s10/2.pgm, 10,
full/s10/3.pgm, 10,
full/s10/4.pgm, 10,
full/s10/5.pgm, 10,
full/s11/1.pgm, 11,
full/s11/2.pgm, 11,
full/s11/3.pgm, 11,
full/s11/4.pgm, 11,
full/s11/5.pgm, 11,
full/s12/1.pgm, 12,
full/s12/2.pgm, 12,
full/s12/3.pgm, 12,
full/s12/4.pgm, 12,
full/s12/5.pgm, 12,
full/s13/1.pgm, 13,
full/s13/2.pgm, 13,
full/s13/3.pgm, 13,
full/s13/4.pgm, 13,
full/s13/5.pgm, 13,
full/s14/1.pgm, 14,
full/s14/2.pgm, 14,
full/s14/3.pgm, 14,
full/s14/4.pgm, 14,
full/s14/5.pgm, 14,
full/s15/1.pgm, 15,
full/s15/2.pgm, 15,
full/s15/3.pgm, 15,
full/s15/4.pgm, 15,
full/s15/5.pgm, 15,
full/s16/1.pgm, 16,
full/s16/2.pgm, 16,
full/s16/3.pgm, 16,
full/s16/4.pgm, 16,
full/s16/5.pgm, 16,
full/s17/1.pgm, 17,
full/s17/2.pgm, 17,
full/s17/3.pgm, 17,
full/s17/4.pgm, 17,
full/s17/5.pgm, 17,
full/s18/1.pgm, 18,
full/s18/2.pgm, 18,
full/s18/3.pgm, 18,
full/s18/4.pgm, 18,
full/s18/5.pgm, 18,
full/s19/1.pgm, 19,
full/s19/2.pgm, 19,
full/s19/3.pgm, 19,
full/s19/4.pgm, 19,
full/s19/5.pgm, 19,
full/s20/1.pgm, 20,
full/s20/2.pgm, 20,
full/s20/3.pgm, 20,
full/s20/4.pgm, 20,
full/s20/5.pgm, 20,
full/s21/1.pgm, 21,
full/s21/2.pgm, 21,
full/s21/3.pgm, 21,
full/s21/4.pgm, 21,
full/s21/5.pgm, 21,
full/s22/1.pgm, 22,
full/s22/2.pgm, 22,
full/s22/3.pgm, 22,
full/s22/4.pgm, 22,
full/s22/5.pgm, 22,
full/s23/1.pgm, 23,
full/s23/2.pgm, 23,
full/s23/3.pgm, 23,
full/s23/4.pgm, 23,
full/s23/5.pgm, 23,
full/s24/1.pgm, 24,
full/s24/2.pgm, 24,
full/s24/3.pgm, 24,
full/s24/4.pgm, 24,
full/s24/5.pgm, 24,
full/s25/1.pgm, 25,
full/s25/2.pgm, 25,
full/s25/3.pgm, 25,
full/s25/4.pgm, 25,
full/s25/5.pgm, 25,
full/s26/1.pgm, 26,
full/s26/2.pgm, 26,
full/s26/3.pgm, 26,
full/s26/4.pgm, 26,
full/s26/5.pgm, 26,
full/s27/1.pgm, 27,
full/s27/2.pgm, 27,
full/s27/3.pgm, 27,
full/s27/4.pgm, 27,
full/s27/5.pgm, 27,
full/s28/1.pgm, 28,
full/s28/2.pgm, 28,
full/s28/3.pgm, 28,
full/s28/4.pgm, 28,
full/s28/5.pgm, 28,
full/s29/1.pgm, 29,
full/s29/2.pgm, 29,
full/s29/3.pgm, 29,
full/s29/4.pgm, 29,
full/s29/5.pgm, 29,
full/s30/1.pgm, 30,
full/s30/2.pgm, 30,
full/s30/3.pgm, 30,
full/s30/4.pgm, 30,
full/s30/5.pgm, 30,
full/s31/1.pgm, 31,
full/s31/2.pgm, 31,
full/s31/3.pgm, 31,
full/s31/4.pgm, 31,
full/s31/5.pgm, 31,
full/s32/1.pgm, 32,
full/s32/2.pgm, 32,
full/s32/3.pgm, 32,
full/s32/4.pgm, 32,
full/s32/5.pgm, 32,
full/s33/1.pgm, 33,
full/s33/2.pgm, 33,
full/s33/3.pgm, 33,
full/s33/4.pgm, 33,
full/s33/5.pgm, 33,
full/s34/1.pgm, 34,
full/s34/2.pgm, 34,
full/s34/3.pgm, 34,
full/s34/4.pgm, 34,
full/s34/5.pgm, 34,
full/s35/1.pgm, 35,
full/s35/2.pgm, 35,
full/s35/3.pgm, 35,
full/s35/4.pgm, 35,
full/s35/5.pgm, 35,
full/s36/1.pgm, 36,
full/s36/2.pgm, 36,
full/s36/3.pgm, 36,
full/s36/4.pgm, 36,
full/s36/5.pgm, 36,
full/s37/1.pgm, 37,
full/s37/2.pgm, 37,
full/s37/3.pgm, 37,
full/s37/4.pgm, 37,
full/s37/5.pgm, 37,
full/s38/1.pgm, 38,
full/s38/2.pgm, 38,
full/s38/3.pgm, 38,
full/s38/4.pgm, 38,
full/s38/5.pgm, 38,
full/s39/1.pgm, 39,
full/s39/2.pgm, 39,
full/s39/3.pgm, 39,
full/s39/4.pgm, 39,
full/s39/5.pgm, 39,
full/s40/1.pgm, 40,
full/s40/2.pgm, 40,
full/s40/3.pgm, 40,
full/s40/4.pgm, 40,
full/s40/5.pgm, 40,
full/s41/1.pgm, 41,
full/s41/2.pgm, 41,
full/s41/3.pgm, 41,
full/s41/4.pgm, 41,
full/s41/5.pgm, 41,