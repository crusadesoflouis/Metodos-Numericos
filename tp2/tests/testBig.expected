1754.876031
1438.322785
1080.001794
957.691923
911.195246
759.177796
695.293039
646.297559
599.166527
538.317909
517.956071
473.802174
459.206288
438.861392
421.206386
